.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Windows API question
+    url: https://stackoverflow.com/questions/tagged/winapi
+    about: Please ask and answer questions about the Windows API on StackOverflow
```

Comment:
Think there's a dedicated support staff for general Windows API development on Q&A, so should we at least try pushing folks that way first?

Reply:
Sounds good; will update!